fp_app/app/controllers/planners_controller.rb

```diff
@@ -1,16 +1,23 @@
 class PlannersController < ApplicationController
-  before_action :authenticate_planner!, only: %i[show edit update]
-  before_action :authenticate_user!, only: %i[index]
-  before_action :correct_planner, only: %i[show edit update]
-
+  before_action :authenticate_planner!, only: %i[ edit update ]
+  before_action :authenticate_user!, only: %i[ index  ]
+  before_action :correct_planner, only: %i[ edit update ]
 
   def index
     @planners = Planner.all
   end
 
   def show
     @planner = Planner.find(params[:id])
-    @appointments = Appointment.where(planner_id: @planner.id)
+
+    if planner_signed_in? && current_planner == @planner
+      @appointments = Appointment.where(planner_id: @planner.id)
```

Comment:
```suggestion
      @appointments = @planner.appointments
```
relationがあると思うので上記の方がより良いと思います

Reply:
ありがとうございます！
申し訳ありません。appointmentのルーティング、機能の実装は次のPRで行っていました。
今回はappointmentsの記述を削除しておきます。
次回のPRで`@appointments = @planner.appointments`に変更しています！